frontend/src/views/camp/Dashboard.vue

```diff
@@ -341,13 +342,20 @@ export default {
     ...mapGetters({
       loggedInUser: 'getLoggedInUser',
     }),
+    /**
+     * Is True until the Component gets unmounted or there is a Navigation to another page
+     */
+    syncUrlQuerActive() {
+      return this.isActive && this.$router.currentRoute.name === 'camp/dashboard'
```

Comment:
This didn't work for me. `this.$router.currentRoute` still equals to the `camp/dashboard` route, even after the Print button was clicked. It seems that `currentRoute` is only updated, once the navigation is completely finished.

What we probably would need is to query for a pending navigation route. In vue-router there's no built-in capability to do this. It seems to be implementable with Navigation Guards, however, might not be trivial to catch all the edge cases (see https://github.com/vuejs/vue-router/issues/2012#issuecomment-735777617).

Reply:
I see your point and used window.history.replaceState - this doesn't overwrite pending navigations.
This could cause problems with mobile devices tho.